### ISSUE TITLE
Why did we remove this definition namespace? 

### ISSUE THREAD

**Issue by congtung10t2** [pixijs/pixi-spine#233]
![capture](https://user-images.githubusercontent.com/10750828/37640120-1fbbc3f6-2c4f-11e8-8a20-b25b03d90379.PNG)

**Comment by congtung10t2** [comment]
I got it in new spine version at next branch. 
When i try to implement. this notify that it can't find PIXI_Spine.

**Comment by ivanpopelyshev** [comment]
oops. sorry :) gonna republish it

**Comment by ivanpopelyshev** [comment]
actually, its here: https://github.com/pixijs/pixi-spine/blob/next/bin/pixi-spine.js#L6614 , its inside the namespace.

**Comment by ivanpopelyshev** [comment]
Can you describe the problem more or give me the fiddle?

**Comment by congtung10t2** [comment]
![capture](https://user-images.githubusercontent.com/10750828/37644251-1b893200-2c5e-11e8-9cb0-972f60f66389.PNG)
this is what i get from console

**Comment by ivanpopelyshev** [comment]
Did you start using pixi@next?

**Comment by congtung10t2** [comment]
Yes. I already used it and got some errors. So far, I'm still finding why? but it's seem you rename a lot of function and class. With current PIXI 4.4.1 on mine , it doesn't work.

**Comment by ivanpopelyshev** [comment]
I made this branch for typings to be compatible with v5: https://github.com/pixijs/pixi.js/tree/next 

The branch doesnt work with 4.4.1 or any other 4.x version.

**Comment by congtung10t2** [comment]
ohh, I see. that may I have decide to upgrade or custom my own version. 
Thanks.
